### DISCUSSION TITLE
How can I prevent conform.nvim from being installed?

### DISCUSSION THREAD

**Discussion by beetol** [LazyVim/LazyVim#4343] [Q&A] [answered]
I have Neovim 0.9.5 and LazyVim notifies me that conform.nvim is being installed and that it's only compatible with Neovim 0.10.

I tried removing it from:

- lazy-lock.json
- .local/share/nvim/lazy. 
- .cache/nvim
- plugins/example.lua

But it still keeps popping up like some zombie.

**Comment by Jomik** [comment, marked as answer]
I assume there is a reason you don't update to 0.10, even though it is stable?
You have to set disable conform explicitly [lazy.nvim/spec](https://lazy.folke.io/spec#spec-loading)
So add
```lua
{
  "stevearc/conform.nvim",
  enabled = false,
}
```
Do note that your formatting will break - unsure if there is other issues.. See [LazyVim/Formatting](https://www.lazyvim.org/plugins/formatting)

**Reply by beetol** [reply]
I've had some frustrating experiences with 0.10 as it broke my plugins when I was using LunarVim (for example, NvimTree wasn't working as expected on 0.10). I didn't realize 0.10 was stable until now.

Thanks for the suggestion!
